internal/adapters/scheduler/scheduler_storage_pg.go

```diff
@@ -114,45 +118,51 @@ func (s schedulerStorage) GetScheduler(ctx context.Context, name string) (*entit
 	return scheduler, nil
 }
 
-func (s schedulerStorage) GetSchedulerWithFilter(ctx context.Context, SchedulerFilter *filters.SchedulerFilter) (*entities.Scheduler, error) {
+func (s schedulerStorage) GetSchedulerWithFilter(ctx context.Context, schedulerFilter *filters.SchedulerFilter) (*entities.Scheduler, error) {
 	client := s.db.WithContext(ctx)
 	var dbScheduler Scheduler
 	queryString := queryGetScheduler
 
 	var queryArr []interface{}
-	if SchedulerFilter.Name == "" {
+	if schedulerFilter.Name == "" {
 		return nil, errors.NewErrInvalidArgument("Scheduler need at least a name")
 	}
-	queryArr = append(queryArr, SchedulerFilter.Name)
+	queryArr = append(queryArr, schedulerFilter.Name)
 
-	if SchedulerFilter.Version != "" {
+	if schedulerFilter.Version != "" {
 		queryString = queryString + " and v.version = ?"
-		queryArr = append(queryArr, SchedulerFilter.Version)
+		queryArr = append(queryArr, schedulerFilter.Version)
 	}
 
 	queryString = queryString + " order by created_at desc limit 1"
+	start := time.Now()
 	_, err := client.QueryOne(&dbScheduler, queryString, queryArr...)
+	reportSchedulerStorageLatency(start, "GetSchedulerWithFilter")
 	if err == pg.ErrNoRows {
-		return nil, errors.NewErrNotFound("scheduler %s not found", SchedulerFilter.Name)
+		return nil, errors.NewErrNotFound("scheduler %s not found", schedulerFilter.Name)
 	}
 	if err != nil {
-		return nil, errors.NewErrUnexpected("error getting scheduler %s", SchedulerFilter.Name).WithError(err)
+		reportSchedulerStorageFailsCounterMetric(schedulerFilter.Name)
```

Comment:
Shouldn't we use the operation name here? (GetSchedulerWithFilter)

Reply:
Yeah